tests/testthat/test-test-threads.R

```diff
@@ -16,6 +18,28 @@ cleanup_bbi <- function(.recreate_dir = FALSE) {
 }
 cleanup_bbi(.recreate_dir = TRUE)
 
+
+get_est_options <- function(search_str = "MAXEVAL|NITER|NBURN", mods){
```

Comment:
Minor and okay if it stays as is, but I don't see the value in having a default for `search_str` given that all spots pass in the value explicitly (which I think is a good thing).